cmd/ci-operator/main.go

```diff
@@ -1495,3 +1483,38 @@ func getPullSecretFromFile(filename string) (*coreapi.Secret, error) {
 	secret.Data[coreapi.DockerConfigJsonKey] = src
 	return secret, nil
 }
+
+func (o *options) getResolverInfo(jobSpec *api.JobSpec) *load.ResolverInfo {
+	// address and variant can only be set via options
+	info := &load.ResolverInfo{
+		Address: o.resolverAddress,
+		Variant: o.variant,
+	}
+
+	allRefs := jobSpec.ExtraRefs
+	if jobSpec.Refs != nil {
+		allRefs = append([]prowapi.Refs{*jobSpec.Refs}, allRefs...)
```

Comment:
Actually -- in the case of a generated periodic, `jobSpec.Refs` will be `nil` but `jobSpec.ExtraRefs` will not and we _do_ want to use the `jobSpec.ExtraRefs` there. Please fix in a follow-up